feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -93,15 +93,17 @@ class PreviewViewModel @AssistedInject constructor(
             combine(
                 cameraUseCase.getCurrentSettings().filterNotNull(),
                 constraintsRepository.systemConstraints.filterNotNull(),
-                cameraUseCase.getZoomScale()
-            ) { cameraAppSettings, systemConstraints, zoomScale ->
+                cameraUseCase.getZoomScale(),
+                cameraUseCase.getLowLightBoostActiveStatus()
```

Comment:
We're going to start to run out of the ability to combine these flows if we keep adding more. `combine` only has overloads for up to 5 flows (with unique types). Since zoomScale and LLBActiveStatus are both descriptive camera state, maybe we can combine them into a single data class called `DescriptiveCameraState`? Then we can have a single flow for both from `CameraUseCase`.